web-bundle/src/main/java/com/graphhopper/replica/GtfsLinkMapper.java

```diff
@@ -202,4 +214,69 @@ public void setGtfsLinkMappings() {
     private static String getRouteInfoString(Route route, String agencyName) {
         return agencyName + "," + route.route_short_name + "," + route.route_long_name + "," + route.route_type;
     }
+
+    // returns all CSV rows (as a list of Strings) derived from a single GTFS feed's data
+    private List<String> getGtfsLinkCsvRowsForFeed(SetMultimap<String, String> routeIdToTripsInRoute,
+                                                   Map<String, List<Pair<Stop, Stop>>> tripIdToStopPairsInTrip,
+                                                   HTreeMap<String, String> gtfsLinkMappings) {
+        List<String> rowsForFeed = Lists.newArrayList();
+        for (String routeId : routeIdToTripsInRoute.keySet()) {
+            for (String tripIdInRoute : routeIdToTripsInRoute.get(routeId)) {
+                for (Pair<Stop, Stop> stopStopPair : tripIdToStopPairsInTrip.get(tripIdInRoute)) {
+                    String stopPairString = stopStopPair.getLeft().stop_id + "," + stopStopPair.getRight().stop_id;
+
+                    // Skip stop-stop pairs where we couldn't find a valid route
+                    if (!gtfsLinkMappings.containsKey(stopPairString)) {
+                        continue;
+                    }
+                    List<String> stableEdgeIds = Lists.newArrayList(gtfsLinkMappings.get(stopPairString).split(","));
+                    String stableEdgeIdString = String.format("\"[%s]\"", stableEdgeIds.stream().map(id -> "'" + id + "'").collect(Collectors.joining(",")));
```

Comment:
todo: output `""` for this if we don't have any stable edge IDs. So, the CSV column will look like `,,` basically